src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -70,6 +72,19 @@ public StorageFile(String filePath) throws InvalidStorageFilePathException {
         }
     }
 
+    public boolean exist() {
+    	File f = new File(DEFAULT_STORAGE_FILEPATH);
+    	if(f.exists()) {
+    		return true;
+    	}
+    	else {
+    		return false;
+    	}
```

Comment:
This if-else block is not necessary, and can be replaced with a single return statement.